### PR TITLE
Fixed non-UTF characters.

### PR DESCRIPTION
**Pull request by fjpanag**
## Summary

Fixes some non-UTF characters that are not displaying properly on my system.

## Impact

N/A

## Testing

N/A

### CODE REVIEW

**Review by pkarashchenko — APPROVED** [review]

File: boards/arm/sama5/jupiter-nano/include/board_sdram.h [hunk 1]
```diff
@@ -107,7 +107,7 @@
 
 #define BOARD_ADC_PRESCAL          (7)
 #define BOARD_TSD_STARTUP          (40)        /* 40 nanoseconds */
-#define BOARD_TSD_TRACKTIM         (2000)      /* Min 1�s at 8MHz */
+#define BOARD_TSD_TRACKTIM         (2000)      /* Min 1µs at 8MHz */
 #define BOARD_TSD_DEBOUNCE         (10000000)  /* 10 milliseconds (units nanoseconds) */
```
Comment (xiaoxiang781216) [inline boards/arm/sama5/jupiter-nano/include/board_sdram.h:111]: can we change to ASCII u?

File: boards/arm/sama5/jupiter-nano/include/board_384mhz.h [hunk 2]
```diff
@@ -102,7 +102,7 @@
  *
  * "The USB Host controller requires 48 MHz and 12 MHz clocks for OHCI
  *  full-speed operations. These clocks must be generated by a PLL with a
- *  correct accuracy of � 0.25% thanks to USBDIV field.
+ *  correct accuracy of ± 0.25% thanks to USBDIV field.
```
Comment (xiaoxiang781216) [inline boards/arm/sama5/jupiter-nano/include/board_384mhz.h:105]: change +-?

File: boards/arm/sama5/jupiter-nano/include/board_sdram.h (hunk 1 above)
Comment (fjpanag) [reply boards/arm/sama5/jupiter-nano/include/board_sdram.h:111]: Generally this is my preference, as I believe that source code should only be ASCII.

But when I raised this concern in the mailing list, it seems that people prefer to use such special symbols.

What do you recommend me to do?

Comment (fjpanag) [reply boards/arm/sama5/jupiter-nano/include/board_sdram.h:111]: OK, perfect!

Fixed.

File: boards/arm/sama5/jupiter-nano/include/board_384mhz.h (hunk 2 above)
Comment (fjpanag) [reply boards/arm/sama5/jupiter-nano/include/board_384mhz.h]: Fixed.

**Comment by fjpanag** [comment]
BTW, this is how our CI checks for invalid characters:

```sh
cvt2utf convert -i h c --nobak .

if [ -n "$(git status -s)" ]; then
	exit 1
else
	exit 0
fi
```

This is very simple, it could be added in NuttX CI.

**Comment by xiaoxiang781216** [comment]
> BTW, this is how our CI checks for invalid characters:
> 
> ```shell
> cvt2utf convert -i h c --nobak .
> 
> if [ -n "$(git status -s)" ]; then
> 	exit 1
> else
> 	exit 0
> fi
> ```
> 
> This is very simple, it could be added in NuttX CI.

could you integrate the check to https://github.com/apache/nuttx/blob/master/tools/checkpatch.sh? So, we don't fix the same problem again.

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by fjpanag** [comment]
> > BTW, this is how our CI checks for invalid characters:
> > ```shell
> > cvt2utf convert -i h c --nobak .
> > 
> > if [ -n "$(git status -s)" ]; then
> > 	exit 1
> > else
> > 	exit 0
> > fi
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This is very simple, it could be added in NuttX CI.
> 
> could you integrate the check to https://github.com/apache/nuttx/blob/master/tools/checkpatch.sh? So, we don't fix the same problem again.

Yes I will do.

However, it requires [cvt2utf](https://github.com/x1angli/cvt2utf) to be installed with:
```sh
pip install cvt2utf
```

How/where can I install this to the CI image?

**Comment by xiaoxiang781216** [comment]
you can install here:
https://github.com/apache/nuttx/blob/master/tools/ci/docker/linux/Dockerfile#L282
https://github.com/apache/nuttx/blob/master/tools/ci/cibuild.sh#L286

**Pull request merged**